Chess/include/Piece.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include <iostream>
+#include <string>
+#include <vector>
+#include "Board.h"
```

Comment:
Remove unnecessary includes: In the "Piece.h" file, the line #include "Board.h" is unnecessary. If it's not required for the functionality of the Piece class, it should be removed to avoid unnecessary dependencies.

Reply:
In the protected members of the Piece class there is a member Board* m_board. 
So the include is necessary in this case.